translations/es.ts

```diff
@@ -1867,6 +1864,86 @@ Será instalada cuando reinicies qTox.</translation>
         <comment>Desktop as a camera input for screen sharing</comment>
         <translation>Pantalla</translation>
     </message>
+    <message>
+        <source>Incorrect responce</source>
+        <translation>Respuesta erronea</translation>
+    </message>
+    <message>
+        <source>No password in response</source>
+        <translation>Respuesta sin contraseña</translation>
+    </message>
+    <message>
+        <source>Server doesn&apos;t support Toxme</source>
+        <translation>El servidor no soporta Toxme</translation>
+    </message>
+    <message>
+        <source>You must send POST requests to /api</source>
+        <translation>Tienes que mandar una petición POST a /api</translation>
+    </message>
+    <message>
+        <source>Please try again using a HTTPS connection</source>
+        <translation>Inténtalo de nuevo usando una conexión HTTPS</translation>
+    </message>
+    <message>
+        <source>I was unable to read your encrypted payload</source>
+        <translation>No fue posible leer tu payload cifrado</translation>
+    </message>
+    <message>
+        <source>You&apos;re making too many requests. Wait an hour and try again</source>
+        <translation>Estás haciendo demasiadas peticiones. Inténtalo de nuevo en una hora</translation>
+    </message>
+    <message>
+        <source>This name is already in use</source>
+        <translation>Ese nombre ya está en uso</translation>
+    </message>
+    <message>
+        <source>This Tox ID is already registered under another name</source>
+        <translation>Esta Tox ID ya fue registrada bajo otro nombre</translation>
+    </message>
+    <message>
+        <source>Please don&apos;t use a space in your name</source>
+        <translation>Espacios en el nombre no están permitidos</translation>
+    </message>
+    <message>
+        <source>Password incorrect</source>
+        <translation>Contraseña incorrecta</translation>
+    </message>
+    <message>
+        <source>You can&apos;t use this name</source>
+        <translation>No puedes usar ese nombre</translation>
+    </message>
+    <message>
+        <source>Name not found</source>
+        <translation>Nombre no encontrado</translation>
+    </message>
+    <message>
+        <source>Tox ID not sent</source>
+        <translation>No se envió la Tox ID</translation>
+    </message>
+    <message>
+        <source>Lookup failed because the other server replied with invalid data</source>
+        <translation>La consulta falló debido a datos invalidos enviados por otro servidor</translation>
```

Comment:
I'm omitting "el" on purpose here, to keep it generic. The user doesn't know which one "the other server" is, how many there are or what do they do. IMO the original gives no relevant info to the average user and should be improved @Diadlo @tux3